src/primitive.js

```diff
@@ -441,7 +448,7 @@ var _ = Mavo.Primitive = class Primitive extends Mavo.Node {
 
 		let editor = this.editor ?? this.originalEditor;
 
-		if (editor?.matches("select:not(.mv-options-select")) {
+		if (editor?.matches("select:not(.mv-options-select)")) {
```

Comment:
Whoa, how did this go undetected all this time?!

Reply:
We have it not so long as you might imagine—not longer than we have `mv-options`. 😉 We couldn't detect it thanks to the browser forgiveness, I believe. I checked in the console—both variants return the same result.